components/StyledLinkSvgContainer/index.js

```diff
@@ -21,14 +20,18 @@ const StyledSpan = styled.span`
   justify-content: center;
 `;
 
-export default function StyledLinkSvgContainer({ dynamicId }) {
-    
-  let href = `/contacts/${dynamicId}`;
+export default function StyledLinkSvgContainer({ dynamicId, svg }) {
+  let href = "";
+  if (svg === infoSVG) {
```

Comment:
As you will always have a value for your svg prop, you could simplify this by using a ternary operator:
```js
const href = (svg === infoSVG) ? `/contacts/${dynamicId}` : `/contacts-call/${dynamicId}`
```

Reply:
According to your proposal I  leave only one "else" instead of "if else". I leave the if because in the future there will be more buttons and the ternary operator you offer will not work with 4 buttons. Thanks for the proposal anyway